storage/connectors/bigquery/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BatchLoadsWithResult.java

```diff
@@ -135,6 +142,12 @@
 
     PCollection<WriteBundlesToFiles.Result<DestinationT>> results =
         input
+            .apply("WindowWithTrigger",
+                Window.<KV<DestinationT, TableRow>>configure()
+                    .triggering(
+                        Repeatedly.forever(
+                            AfterPane.elementCountAtLeast(FILE_TRIGGERING_RECORD_COUNT)))
```

Comment:
The trigger still happens after some amount of time regardless of whether this count is reached, right?

Reply:
yes, end-of-window trigger is implied here